jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/navigation/StepperTests.java

```diff
@@ -13,40 +13,40 @@
     public void mainStepperTest() {
         navigationStepperHorizontalLinearStepperPage.open();
         stepperFrame.stepper.is().stepperDisplayed();
-        stepperFrame.stepper.is().stepTitleDisplayed("You are on Step #1");
-        stepperFrame.stepper.clickNextButton();
-        stepperFrame.stepper.is().stepTitleDisplayed("You are on Step #2");
-        stepperFrame.stepper.clickNextButton();
-        stepperFrame.stepper.is().stepTitleDisplayed("You are on Step #3");
-        stepperFrame.stepper.clickFinishButton();
-        stepperFrame.stepper.is().stepTitleDisplayed("All steps completed");
-        stepperFrame.stepper.clickResetButton();
-        stepperFrame.stepper.is().stepTitleDisplayed("You are on Step #1");
-        stepperFrame.stepper.clickNextButton();
-        stepperFrame.stepper.clickBackButton();
-        stepperFrame.stepper.is().stepTitleDisplayed("You are on Step #1");
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #1");
+        stepperFrame.stepperNextButton.clickNextButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #2");
+        stepperFrame.stepperNextButton.clickNextButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #3");
+        stepperFrame.stepperFinishButton.clickFinishButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("All steps completed");
+        stepperFrame.stepperResetButton.clickResetButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #1");
+        stepperFrame.stepperNextButton.clickNextButton();
+        stepperFrame.stepperBackButton.clickBackButton();
+        stepperFrame.stepperTitle.is().stepTitleDisplayed("You are on Step #1");
```

Comment:
Update name of the method' stepTitleDisplayed' to' text()' or something like this, as word displayed has association with only display on page without checking for text

Reply:
done